core-services/ci-secret-bootstrap/_config.yaml

```diff
@@ -917,6 +917,45 @@ secret_configs:
     - non_app_ci
     name: cluster-secrets-openstack-vexxhost
     namespace: ci
+- from:
+    .awscred:
+      attachment: .awscred
+      bw_item: openstack
+    insights-live.yaml:
+      attachment: insights-live.yaml
+      bw_item: insights-ci-account
+    pull-secret:
+      dockerconfigJSON:
+      - auth_bw_attachment: token_image-puller_app.ci_reg_auth_value.txt
+        bw_item: build_farm
+        registry_url: registry.ci.openshift.org
+      - auth_bw_attachment: auth
+        bw_item: cloud.openshift.com-pull-secret
+        email_bw_field: email
+        registry_url: cloud.openshift.com
+      - auth_bw_attachment: auth
+        bw_item: quay.io-pull-secret
+        email_bw_field: email
+        registry_url: quay.io
+      - auth_bw_attachment: auth
+        bw_item: registry.connect.redhat.com-pull-secret
+        email_bw_field: email
+        registry_url: registry.connect.redhat.com
+      - auth_bw_attachment: auth
+        bw_item: registry.redhat.io-pull-secret
+        email_bw_field: email
+        registry_url: registry.redhat.io
+    ssh-privatekey:
+      attachment: ssh-privatekey
+      bw_item: jenkins-ci-provisioner
+    ssh-publickey:
+      attachment: ssh-publickey
+      bw_item: jenkins-ci-provisioner
+  to:
+  - cluster_groups:
+    - non_app_ci
+    name: cluster-secrets-openstack-kuryr
+    namespace: ci
```

Comment:
This needs to be submitted as its own patch. 
It is not an emediate operation because it has to propagate to all the ci clusters. 
So split this PR and submit the changes to this file by itself. 

After the changes to this file merge, (about 12 hours after)   the secret "cluster-secrets-openstack-kuryr" will be available, 

then you just have to set CLUSTER_TYPE=kuryr and

Reply:
Got it, thanks! I will split it